PyPoE/cli/exporter/wiki/parsers/skill.py

```diff
@@ -751,15 +751,14 @@ def export(self, parsed_args, skills):
                 self._skill(ge=skill, infobox=data, parsed_args=parsed_args)
             except Exception as e:
                 console(
-                    'Error when parsing skill "%s":' % skill['Id'],
+                    'Error when parsing skill "{}" at {}:'.format(skill['Id'], skill.rowid),
```

Comment:
[same](https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37/files#r744102703)